src/addons/Portal/Portal.js

```diff
@@ -37,15 +37,27 @@ class Portal extends Component {
     /** Controls whether or not the portal should close on blur of the trigger. */
     closeOnTriggerBlur: PropTypes.bool,
 
-    /** Controls whether or not the portal should close when mousing out of the trigger. */
-    closeOnTriggerMouseLeave: PropTypes.bool,
+    /** Controls whether or not the portal should close on blur of the trigger. */
+    closeOnTriggerClick: PropTypes.bool,
+
+    /**
+     * Controls whether or not the portal should close when mousing out of the
+     * trigger OR the portal content.
+     */
+    closeOnMouseLeave: PropTypes.bool,
```

Comment:
Hm, seems like we should separate these.  I know the names are hideous, but I can see wanting one or the other:

`closeOnTriggerMouseLeave`
`closeOnPortalMouseLeave`

Thoughts?